### PR TITLE
feat: add checkbox to allow updating of caps with vendor prefixes

### PR DESCRIPTION
**Pull request by jlipps**
This should help folks migrate to Appium 2.0 by automatically updating their caps with a prefix unless they deselect the checkbox.

### CODE REVIEW

File: app/renderer/actions/Session.js [hunk 1]
```diff
@@ -174,16 +179,33 @@ export function removeCapability (index) {
   };
 }
 
+function addVendorPrefixes (caps, dispatch, getState) {
+  const newCaps = caps.map((cap) => {
+    // if we don't have a valid unprefixed cap or a cap with an existing prefix, update it
+    if (!VALID_W3C_CAPS.includes(cap.name) && cap.name.split(':').length < 2) {
```
Comment (mykola-mokhnach) [inline app/renderer/actions/Session.js:185]: the second check could also be represented as `_.includes(cap.name, ':')`

File: app/renderer/actions/Session.js [hunk 2]
```diff
@@ -174,16 +179,33 @@ export function removeCapability (index) {
   };
 }
 
+function addVendorPrefixes (caps, dispatch, getState) {
+  const newCaps = caps.map((cap) => {
```
Comment (mykola-mokhnach) [inline app/renderer/actions/Session.js:183]: prefixedCaps

File: app/renderer/components/Session/NewSessionForm.js [hunk 3]
```diff
@@ -98,7 +99,17 @@ export default class NewSessionForm extends Component {
...
             </Row>)}
             <Row>
-              <Col span={24}>
+              <Col span={22}>
+                <FormItem>
+                  <Checkbox
+                    onChange={(e) => setAddVendorPrefixes(e.target.checked)}
+                    checked={addVendorPrefixes}
+                  >
+                    {t('Automatically add necessary Appium vendor prefixes on start')}
```
Comment (mykola-mokhnach) [inline app/renderer/components/Session/NewSessionForm.js:108]: I assume this string should be also added into `en` resources

File: app/renderer/reducers/Session.js [hunk 4]
```diff
@@ -64,6 +64,8 @@ const INITIAL_STATE = {
   isValidCapsJson: true,
   isValidatingCapsJson: false,
   isAddingCloudProvider: false,
+
```
Comment (mykola-mokhnach) [inline app/renderer/reducers/Session.js:67]: extra space here

**Review by KazuCocoa — APPROVED** [review]
lg (Some comments by mykola were same as I thought)

File: app/renderer/components/Session/NewSessionForm.js (hunk 3 above)
Comment (jlipps) [reply app/renderer/components/Session/NewSessionForm.js:108]: https://github.com/appium/appium-desktop/pull/1859

**Pull request merged**